sigle/src/modules/userCard/UserCard.tsx

```diff
@@ -67,6 +69,9 @@ export const UserCard = ({ address, userFollowing }: UserCardProps) => {
       return names.names[0];
     }
   );
+  const { data: userFollowing } = useGetGaiaUserFollowing({
+    enabled: !!user && username !== user.username,
```

Comment:
let's use data from the cache here so we don't fetch it for every list item when it's in a big list.

Reply:
okay sure, should we do the same as we did on the hover card and set staleTime to Infinity?

Reply:
Yup 👍